src/components/search/results/mobile-partial-results.vue

```diff
@@ -29,7 +29,7 @@
       <SlidingPanel
         :resetOnContentChange="false"
         buttonClass="x-button-lead x-button-circle x-button-ghost x-padding--00"
-        class="x-sliding-panel--show-buttons-on-hover"
+        class="x-sliding-panel-show-buttons-on-hover"
       >
         <div class="x-list x-list--gap-05 x-padding--left-05">
```

Comment:
**issue:** `partial-results` are broken in mobile, they're displayed in a column instead of a row.

```suggestion
        <div class="x-list x-list--horizontal x-list--gap-05 x-padding--left-05">
```